sql-plugin/src/main/330/scala/com/nvidia/spark/rapids/v2/RapidsShuffleManager.scala

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright (c) 2021-2022, NVIDIA CORPORATION.
```

Comment:
do we need all these individual shuffle manager files, are some the same that we could put in a 3XXuntil3YY?

Reply:
also no 2021

Reply:
We may be able to. I wasn't sure if I should do it in this PR but sounds like from your earlier comment that you want everything done in one release. I can look into this